Protos/V1/auth_common.proto

```diff
@@ -0,0 +1,31 @@
+/**
+ * Messages describing authentication and associated requests and responses.
+ */
+syntax = "proto3";
+
+package armonik.api.grpc.v1.auth;
+
+option csharp_namespace = "ArmoniK.Api.gRPC.V1.Auth";
+
+/**
+ * A user.
+ */
+message User {
+  string username = 1; /** Username. If authentication is disabled, must be set to 'Anonymous' */
+
+  repeated string roles = 2; /** Roles. If authentication is disabled, must return []. */
+  repeated string permissions = 3; /** Permissions. If authentication is disabled, must return every permissions. */
+}
+
+
+/**
+ * Request to get active user informations.
+ */
+message GetMeRequest {}
+
+/**
+ * Response to get active user informations.
+ */
+message GetMeResponse {
+  User user = 1; /** Return active user. */
```

Comment:
Should be a OneOf between User and an error, in case authentication is enabled but the certificate or user isn't in the database. Since this endpoint shouldn't be checked for authentication (Maybe ?)

Reply:
The error will be thrown by gRPC so data will be the error not the response.